### PR TITLE
Fix Typos in interact

### PR DESCRIPTION
**Pull request by vladimirfomene**
Fix Typos in interact

## Description

Change from "permissionaly intract" to "permissively interact"

### CODE REVIEW

**Comment by ryancreatescopy** [comment]
@samajammin could this be made even clearer to say – "interact with, without needing permission"?

File: src/content/developers/docs/ethereum-stack/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ As a dapp developer, you don't need to know much about the EVM other than it exi
...
 
-Not only do smart contracts serve as open source libraries, they are essentially open API services that run 24/7 and can't be taken down. Smart contracts provide public functions which applications ([dapps](/developers/docs/dapps/)) may permissionaly intract with. Any application may integrate with deployed smart contracts to compose functionality (such as data feeds or decentralized exchanges). Anyone can deploy new smart contracts to Ethereum in order to add custom functionality to meet their application's needs.
+Not only do smart contracts serve as open source libraries, they are essentially open API services that run 24/7 and can't be taken down. Smart contracts provide public functions which applications ([dapps](/developers/docs/dapps/)) may permissively interact with. Any application may integrate with deployed smart contracts to compose functionality (such as data feeds or decentralized exchanges). Anyone can deploy new smart contracts to Ethereum in order to add custom functionality to meet their application's needs.
```
Comment (samajammin) [inline src/content/developers/docs/ethereum-stack/index.md:28]:
```suggestion
Not only do smart contracts serve as open source libraries, they are essentially open API services that run 24/7 and can't be taken down. Smart contracts provide public functions which applications ([dapps](/developers/docs/dapps/)) may interact with, without needing permission. Any application may integrate with deployed smart contracts to compose functionality (such as data feeds or decentralized exchanges). Anyone can deploy new smart contracts to Ethereum in order to add custom functionality to meet their application's needs.
```

**Review by samajammin — CHANGES_REQUESTED** [review]
Hey @vladimirfomene - thanks for catching this typo & suggesting this fix!

I suggested a change which I think better solves this issue. As @ryancreatescopy pointed out, your change to `permissively interact with` isn't actually what we're trying to say here. The intention is that they they do not need permission. 

Please accept the change (or let us know what you think) & we can get this merged. Thanks again!

**Review by samajammin — APPROVED** [review]
Looks like I was able to accept the suggested change (didn't know Github allowed that!) so I will accept & merge this PR. Thanks again @vladimirfomene!

**Comment by samajammin** [comment]
@all-contributors please add @vladimirfomene for bug reports & content.

**Pull request merged**